pkgs/by-name/ro/rockyou/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, fetchFromGitLab
+, stdenvNoCC
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "rockyou";
+  version = "unstable-2023-05-12";
```

Comment:
Oh I missed that it is also in SecLists. I'll try to figure something out, doing some sort of package alias that just references seclists and only copies rockyou is probably the best way to go about it, it might take me some time to come back to this since my backlog is currently very long, sorry :/

Reply:
The rockyou lists included by seclists aren't identical to the one from the kali package, the kali one is a lot larger (479k vs 140M) so I would leave it as is.

Reply:
lmao, nevermind my comment about rockyou always being rockyou...

Reply:
No wait, I had another look into this, curious to find out what kind of subset SecLists had used. Were you looking at the rockyou in parts? Because there is this one: https://github.com/danielmiessler/SecLists/blob/395c945627196b6172e6e83a014c964f805e8d48/Passwords/Leaked-Databases/rockyou.txt.tar.gz

Reply:
Oh I missed that I only looked at the largest txt rockyou files